### DISCUSSION TITLE
New dependency to crc32_z(), which requires zlib 1.2.9

### DISCUSSION THREAD

**Discussion by Jens0mat** [rui314/mold#1325] [General]
Quick question please,
In 2.33.0, there is a new dependency to crc32_z() which requires zlib 1.2.9.
I have some older systems that do not yet offer it.
Would it be possible to use crc32() instead?
Not sure if it is a bug, so I use this forum.
Many thx,
Jens

**Comment by rui314** [comment]
Yes, we can use `crc32` instead of `crc32_z` for our use case. I made a change so that mold can compile with older version of zlib. Please check out the git head.

**Comment by Jens0mat** [comment]
Thank you very much for the fast reaction!
I confirm it works for me with commit [1d24b70](https://github.com/rui314/mold/commit/1d24b7099670625e5baa15d35477c89cc32e61cd)!
